src/content/libraries/index.md

```diff
@@ -46,9 +46,9 @@ consult the [Dart API reference.][Dart API]
 : I/O for programs that can use the Dart VM,
   including Flutter apps, servers, and command-line scripts.
 
-[dart:html](/libraries/dart-html)
+[dart:js_interop](/interop/js-interop)
 : DOM and other APIs for browser-based apps.
```

Comment:
I believe the library doesn't really include any DOM APIs (since those are now in `package:web`).

Perhaps we can refer back to our usage of "web platform"?

```suggestion
: APIs for interop with the web platform.
```

Could even cross-link web platform to https://dart.dev/overview#platform if you think that's helpful.